### PR TITLE
More Java sonar fixes

### PR DESCRIPTION
**Pull request by antares1470**
### Description of the Change

As suggested by sonar:
- moved some interface constants to enums
- added default cases to switch statements
- ensured conditionally executed code was reachable

### CODE REVIEW

File: CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/renderer/GLRenderable.java [hunk 1]
```diff
@@ -45,9 +45,21 @@
...
-    public static int DEFAULT_PRIORITY = 10;
-    public static int HIGH_PRIORITY = 0;
+    public enum RenderablePriority {
+        ANNOTATIONS_PRIORITY(20),
+        DEFAULT_PRIORITY(10),
+        HIGH_PRIORITY(0);
+        
+        private final int value;
+        
+        private RenderablePriority(int value) {
```
Comment (cygnus-x-1) [inline CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/renderer/GLRenderable.java:55]: final int

File: CoreUtilities/src/au/gov/asd/tac/constellation/utilities/visual/VisualOperation.java [hunk 2]
```diff
@@ -45,10 +45,22 @@
...
-    public static final int DEFAULT_VISUAL_PRIORITY = 30;
+    public enum VisualPriority {
+        REFRESH_PRIORITY(0),
+        SIGNIFY_IDLE_PRIORITY(10),
+        ELEVATED_VISUAL_PRIORITY(20),
+        DEFAULT_VISUAL_PRIORITY(30);
+        
+        private final int value;
+        
+        private VisualPriority(int value) {
```
Comment (cygnus-x-1) [inline CoreUtilities/src/au/gov/asd/tac/constellation/utilities/visual/VisualOperation.java:56]: final int

File: CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/templates/WorkflowQueryPlugin.java [hunk 3]
```diff
@@ -246,7 +246,7 @@ protected void execute(final PluginGraphs graphs, final PluginInteraction intera
                 } catch (PluginException ex) {
                     wholeOfWorkflowExceptions.add(ex);
                     error = true;
-                    throw ex;
```
Comment (cygnus-x-1) [inline CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/templates/WorkflowQueryPlugin.java:249]: wow - how did we miss this?

File: CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/renderer/GLRenderable.java (hunk 1 above)
Comment (antares1470) [reply CoreOpenGLDisplay/src/au/gov/asd/tac/constellation/visual/opengl/renderer/GLRenderable.java:55]: Done

File: CoreUtilities/src/au/gov/asd/tac/constellation/utilities/visual/VisualOperation.java (hunk 2 above)
Comment (antares1470) [reply CoreUtilities/src/au/gov/asd/tac/constellation/utilities/visual/VisualOperation.java:56]: Done

**Comment by antares1470** [comment]
@arcturus2 @cygnus-x-1 ready for another check

**Pull request merged**